packages/dev-server/package.json

```diff
@@ -19,9 +19,11 @@
   "repository": "andywer/webpack-blocks",
   "bugs": "https://github.com/andywer/webpack-blocks/issues",
   "dependencies": {
-    "webpack": "^2.6.1",
     "webpack-dev-server": "^2.4.5"
   },
+  "peerDependencies": {
+    "webpack": "2.x || 3.x"
+  }
```

Comment:
Missing comma. Would have fixed it myself, but seems like you didn't grant me permission to edit files in the PR 😉